_sass/layout/_search.scss

```
@@ -0,0 +1,23 @@
+.search__wrapper {
+  margin-top: 20px;
+  margin-bottom: 50px;
+}
+
+.searchbar {
+  margin-bottom: 10px;
+
+  input {
+    height: 20px;
+  }
+}
+.ais-Pagination {
+  margin-top: 20px;
+}
```

Adding a note about these `ais` classes can be helpful for other team members to understand what these classes are specifically doing.

```suggestion

// NOTE: These `ais` classes override the defaults from Algolia Instant Search
.ais-Pagination {
  margin-top: 2 * $spacing-base;
}
```